.github/workflows/build_publish.yaml

```diff
@@ -24,11 +25,29 @@ jobs:
         uses: actions/setup-java@v1
         with:
           java-version: 1.8
-          server-id: nexus-sonatype
-          server-username: NEXUS_USERNAME
-          server-password: NEXUS_PASSWORD
       - name: Build and deploy with Maven
-        run: ./mvnw --no-transfer-progress clean -U deploy
+        run: ./mvnw --no-transfer-progress clean -U package -DskipTests
+        
+      - name: List files in target directory
+        run: ls -al omod/target
```

Comment:
@karimshaik786 is this step required in the workflow ? if yes, can you describe the significance